requirements.txt

```diff
@@ -1,3 +1,26 @@
+archapp
+bluesky >=1.6.0
+coloredlogs
+cookiecutter >=1.6.0
+happi >=1.8.0
 ipython
+jinja2
+lightpath >=0.3.0
+matplotlib
+nabs >=0.1.0
+pcdscalc >=v0.2.0
+pcdsdaq >=2.3.0
+pcdsdevices >=3.1.0
+pcdsutils >=0.4.0
+psdm_qs_cli >=0.3.1
+pyfiglet
+pyqt5
 pyyaml
-coloredlogs
+requests
+simplejson
+
+# Some are unavailable on pypi:
+# elog
```

Comment:
Do we need to make this an optional dependency?

Reply:
I think it'd be not so difficult to do. Would you be OK with me taking a stab at that in this PR?

Reply:
Scratch that. I think it deserves a follow-up PR.